compiler/arm/codegen/OMRCodeGenerator.hpp

```diff
@@ -126,7 +126,7 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
    void doRegisterAssignment(TR_RegisterKinds kindsToAssign);
    void doBinaryEncoding();
 
-   void emitDataSnippets(bool isWarm = 0);
+   void emitDataSnippets();
    bool hasDataSnippets();
    int32_t setEstimatedLocationsForDataSnippetLabels(int32_t estimatedSnippetStart, bool isWarm = 0);
```

Comment:
Yes.  There are another batch of functions that need such cleanup (and I have in a subsequent commit).  I was staging these commits as there is surrounding logic that needs to be reasoned about and cleaned up.